README.md

```diff
@@ -79,4 +79,22 @@ customization MUI styles - default font is Inter, primary color
 
 #### commonStyles.tsx -
 
-file with generic/common app styles (Titles, buttons etc.)
\ No newline at end of file
+file with generic/common app styles (Titles, buttons etc.)
+
+### `Local development`
+
+For convinient development locally was created and set up docker compose file. It's a solution also to avoid CORS error.
+
+#### Important! 
+Repos Web FE and Auth should be located in the same directory, otherwise you need to adapt values `volumes` in docker-compose file accordingly.
+
```

Comment:
We need Auth in the mobile app too, so we will keep it together with the FE and adapt values for Mobile App? Is it possible? OR create values for both and leave it outside of both apps?

Reply:
if you check paths in `values` you can see two repos should be located in the same directory. That's all. For the mobile - we need the same file there

Reply:
Ok, I don't really understand what is going on here, so I'll trust you lol